src/Cli/dotnet/commands/dotnet-workload/WorkloadInfoHelper.cs

```diff
@@ -21,6 +23,7 @@ internal class WorkloadInfoHelper : IWorkloadInfoHelper
         private readonly string _targetSdkVersion;
 
         public WorkloadInfoHelper(
+            ParseResult parseResult,
```

Comment:
it might be ok just to pass `bool interactive` instead of whole parse result.

Reply:
fixed